packages/core/lib/debug/interpreter.js

```diff
@@ -50,12 +50,14 @@ class DebugInterpreter {
     const breakpoints = this.session.view(controller.breakpoints);
 
     const currentNode = currentLocation.node ? currentLocation.node.id : null;
-    const currentLine = currentLocation.sourceRange
-      ? currentLocation.sourceRange.lines.start.line
-      : null;
     const currentSourceId = currentLocation.source
       ? currentLocation.source.id
       : null;
+    const currentLine =
+      currentSourceId != null //!= is deliberate
```

Comment:
instead of the comment just do `currentSourceId !== null && currentSourceId !== undefined`, no?

Reply:
Sure, that would also work, if you think that's clearer.